types/fs-extra/OTHER_FILES.txt

```diff
@@ -0,0 +1 @@
+esm.d.mts
```

Comment:
I'm not sure if that is required, if proper ESM import would be used in module test

Reply:
pls disregard, still required when testing TS configs